apps/jupyter/extensions/notebook-runner/src/commands/assign.ts

```diff
@@ -0,0 +1,131 @@
+import { JupyterFrontEnd } from "@jupyterlab/application";
+import { INotebookTracker } from "@jupyterlab/notebook";
+import { ContentsManager, Contents } from "@jupyterlab/services";
+import { NotebookRunnerState } from "../notebook-runner-state";
+import { runAssignCommand } from "../command";
+import { EmbeddedPythonCallbacks } from "../iframe-api";
+import { writeJsonToVirtualFilesystem } from "../utils";
+
+export const registerAssignCommand = (
+  state: NotebookRunnerState,
+  app: JupyterFrontEnd,
+  notebookTracker: INotebookTracker,
+  contentsManager: ContentsManager,
+): void => {
+  app.commands.addCommand(runAssignCommand, {
+    label: "Run Assign",
+    execute: async () => {
+      console.debug(`Waiting for otter session kernel to be available`);
+      const kernel = await state.getOtterKernel();
+
+      console.debug(`Generating student task and autograder..`);
```

Comment:
```suggestion
      console.debug(`Generating student task and autograder...`);
```

1 dot = full stop
3 dots = ellipsis

2 dots = 404 not found :-P